README.md

```diff
@@ -59,6 +80,24 @@ largest application performance monitoring providers) relies on TimescaleDB
 as its main metrics database. TimescaleDB is also the preferred (recommended)
 backend datasource for Zabbix users and is natively supported in Grafana.
 
+## 🌟 Features
+
+Click the video below for an overview of Promscale:
+
+<p align="center">
+<a href="https://www.youtube.com/watch?v=FWZju1De5lc&list=PLsceB9ac9MHTrmU-q7WCEvies-o7"> <img src="https://media.giphy.com/media/aLIuNJwflN425UvYjR/giphy.gif"> </a>
+</p>
+
+* **Analysis in PromQL and SQL** get the ability to analyze data in both query languages. Use PromQL for monitoring and alerting and
+  SQL for deeper analytics and compatibility with a huge ecosystem of data visualization, analysis, and AI/ML tools.
+* **Rock-solid stability** due to being built on top of PostgreSQL, with 30+ years of development work.
+* **Support for backfilling** to ingest data from the past.
+* **High-Availability** support for [Promethus HA deployments](docs/high-avaliability/prometheus-HA.md) as well as
+  high-availability deployments of [TimescaleDB itself](https://blog.timescale.com/blog/high-availability-timescaledb-postgresql-patroni-a4572264a831/).
+* **Simplified architecture**. Unlike some other long-term stores, our architecture consists of only three components: Prometheus, Promscale, and TimescaleDB.
```

Comment:
Should be Simple, not simplified I think, unless there was a more complex architecture we're moving away from.